packages/common/src/utils/stream-utils.ts

```diff
@@ -128,6 +128,9 @@ export class StreamUtils {
     if (state.next?.metadata?.model) {
       cloned.next.metadata.model = state.next.metadata.model.toString()
     }
+    if (state.metadata?.unique && state.metadata.unique instanceof Uint8Array) {
+      cloned.metadata.unique = Array.from(state.metadata.unique)
```

Comment:
Maybe use `uint8arrays` to encode it as base64 to be more compact?

Reply:
This is what we do with CACAO and `linkedBlock` when we serialize them

Reply:
Okay done, but heads up that required me to change behavior based on whether the stream is a TileDocument or not, to avoid changing the type of 'unique' in TileDocument as well, which required me to duplicate the type id number for tile in StreamUtils since I can't import TileDocument from the `common` package.  New PR up if you want to take a look